homework/homework1/task2/wc.py

```diff
@@ -0,0 +1,50 @@
+import sys
+from dataclasses import dataclass
+from typing import TextIO
+
+
+@dataclass
+class FileInformation:
+    file_name: str
+    number_of_lines: int = 0
+    number_of_words: int = 0
+    number_of_bytes: int = 0
+
+    def __str__(self):
+        return f"{self.number_of_lines} {self.number_of_words} {self.number_of_bytes} {self.file_name}"
+
+
+def get_file_information(file_stream: TextIO):
+    info = FileInformation(file_stream.name)
+
+    for line in file_stream:
+        info.number_of_lines += 1
+        info.number_of_words += len(line.split())
+        info.number_of_bytes += len(bytes(line, encoding="utf8"))
+
+    return info
+
+
+def main(*files: str):
+    total = FileInformation("total")
+
+    for file in files:
+        try:
+            with open(file, "r") as f:
+                info = get_file_information(f)
+                total.number_of_lines += info.number_of_lines
+                total.number_of_words += info.number_of_words
+                total.number_of_bytes += info.number_of_bytes
+                print(info)
```

Comment:
Let's add `\t` into the print, so that the numbers are beautifully placed under each other.